src/connection/BaseConnectionOptions.ts

```diff
@@ -47,6 +47,11 @@ export interface BaseConnectionOptions {
      */
     readonly migrationsTableName?: string;
 
+    /**
+     * Transaction mode for migrations to run in
+     */
+    readonly migrationsTransaction?: "all" | "none" | "each";
```

Comment:
maybe rename it so it can be clear that its a "mode" and not a "transaction"?

Reply:
Yep, it is reasonable. Updated the naming